.github/workflows/artifacts.yml

```diff
@@ -0,0 +1,40 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+name: PR Artifacts
+
+on:
+  pull_request:
+    branches: '*'
+
+jobs:
+  docs:
+    name: "Docs Preview"
+    runs-on: "ubuntu-latest"
+    env:
+      GCP_PROJECT_ID: "dev-infra-273822"
+      GCP_BUCKET_ID: "icu4x-pr-artifacts"
+    steps:
+    - uses: actions/checkout@v2
+    - name: Load the default Rust toolchain via the rust-toolchain file.
+      run: rustup show
+    - name: Authenticate to Google Cloud
+      uses: google-github-actions/setup-gcloud@master
```

Comment:
suggestion (non-blocking): change `master` to `v0.2`. It looks like they're following the Github Actions convention to have `v0.2` point to the latest in the `v0.2.x` line (which is currently `v0.2.1`).

Reply:
Good catch!  Done.